rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+Once approved, the Roadmap Shepherd will follow the normal RFC process for [approved RFC(s)](https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0001-rfc-process.md#approving-rfcs).
+
+### Updating the Roadmap
+
+The Roadmap MAY be updated with Project Lead consensus and Oversight approval for a number of reasons including but not limited to:
+
+* A Theme is to be added or deleted.
+* Additions or revision to Objectives change the original intent of the Theme.
+* Emerging conditions require substantive changes to the original Roadmap direction.
+
+Requests to update the Roadmap MUST be forwarded to the Roadmap Shepherd with a detailed description of the proposed changes, including the rationale for the updates, a timeline, and priority.
+
+The Roadmap Shepherd MUST add the request to the next Project Leads meeting agenda for review.
+
+The Roadmap Shepherd and Project Leads MUST review the update request and determine if there is rough consensus.
+
+If consensus is reached, the Roadmap Shepherd MUST refresh the Roadmap and send to Oversight. 
+
+Oversight MUST review the changes and approve.
+
+The Roadmap Shepherd MUST update the previously published Roadmap RFC in the *dcp-community* repo and announce its approval in a similar fashion to the original RFC approval.
+
+## Detailed Design for Planning and Execution
+
+### ZenHub for Project Management
+
+The [ZenHub](https://www.zenhub.com/) project management application has been partially adopted in the DCP community. This RFC fully commits the community to ZenHub as our *one source of truth* for project management and engineering plans.
+
+DCP SHOULD NOT duplicate the ZenHub DCP Board in other documents or spreadsheets. The Board and its reports represent the definitive list of DCP work items for a Release and their current state. 
+
+### DCP Product Backlog
+
+The DCP ZenHub board is the Product Backlog for DCP. All DCP component repos that will assign items to a DCP Release and Milestone MUST be added to the ZenHub *DCP Backlogs* workspace. The [DCP PM Team](https://github.com/orgs/HumanCellAtlas/teams/dcp-pm-team) in the Human Cell Atlas GitHub organization MUST have write access to all the repo(s) in the *DCP workspace*.
+
+The top-level *dcp* repo MUST contain **all** significant work that is important to the DCP community as a whole and require visibility and prioritization. This includes Roadmap objectives and product-wide technical requirements (e.g. “components should have logs...“). 
+
+If Institutions are using other project management applications such as Jira for internal requirements and reporting, then all DCP related issues in a Release MUST be *projected* into ZenHub for transparency and finer granularity for tracking progress.
+
+To avoid maintenance overhead due to duplicate or incomplete issues, the creation of new items in the *dcp* repo SHOULD be limited to the PM Team or a facilitator from the Architecture Team.
```

Comment:
This erects a process barrier to actually making the DCP board reflect what's going on for too little gain.

Reply:
I intend to leave the _SHOULD_ in place. Sam Pierson, Trevor, and I invested many hours cleaning up the board due to poor discipline. We want to avoid maintenance overhead.

Reply:
How about saying SHOULD be limited to PM Team or Tech Leads, or is that the source of the problem?

Reply:
_Too many cooks in the kitchen_ resulted in **almost but not entirely unlike** issues that then needed to be reviewed, merged and closed or laundry lists of future ideas that needed to be triaged, etc. Buy @sampierson a beer and ask him.